packaging/docker/run.sh

```diff
@@ -40,6 +40,13 @@ if [ -n "${PGID}" ]; then
   usermod -a -G "${PGID}" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group docker with ID ${PGID}"
 fi
 
+if mountpoint -q /etc/netdata; then
+  if [ -n "${NETDATA_COPY_STOCK_CONIG}" ] || [ "$(find /etc/netdata -type f | wc -l)" -eq 0 ]; then
+    echo "Copying stock configuration to /etc/netdata"
+    cp -a /etc/netdata.stock /etc/netdata
```

Comment:
```suggestion
    cp -a /etc/netdata.stock/. /etc/netdata
```
We need to copy the content, not the dir.